.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,7 +10,11 @@
 
 - [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
 - [ ] :arrow_left: changes are compatible with RTL direction
-- [ ] :wheelchair: changes are accessible and [tested locally](./../README.md#accessibility-testing)
+- [ ] :wheelchair: If the code change affects the appearance or function of UI,
```

Comment:
Small nit picks:

- simplifying the first sentence to match the other checks;
- having everything in one line;

```suggestion
- [ ] :wheelchair: changes to the UI are tested for accessibility and compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/). You can leverage the built-in [automated testing](./../README.md#accessibility-testing) for basic checks and supplement that with manual testing. Tools like [axe DevTools](https://www.deque.com/axe/devtools/), screen readers e.g. [VoiceOver](https://www.apple.com/voiceover/info/guide/_1121.html), [contrast checkers](https://webaim.org/resources/contrastchecker/) etc. can assist your testing.
```

I'd also suggest extracting the info about testing and tools to an accessibility section in the `README.md` file and link it here - just to keep this checklist as simple and minimal as possible. What do you think?